content/en/about/supported-releases/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Supported Releases
+description: The current supported Istio releases.
+weight: 35
+icon: cadence
+---
+
+The status of currently supported releases.
+
+## Supported versions of Istio
+
+|     | Curently Supported | Release Date    | End of Life       |
+|-----|--------------------|-----------------|-------------------|
+| 1.7 | Yes                | August 21, 2020 |                   |
+| 1.6 | Yes                | May 21, 2020    | November 21, 2020 |
+| 1.5 | No                 |                 |                   |
+| 1.4 | No                 |                 |                   |
+
+### Contains known CVEs
```

Comment:
How should a customer use this? I'm assuming by design any release will have a CVE in few months. Should we add the opposite i.e. releases without Known CVEs?

Reply:
I like your way better. Changed.